recipes/rgi/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# copy MetaGeneMark key to working directory
+cp ./package/rgi/mgm/gm_key ~/.gm_key
```

Comment:
This does not work or does not have any influence. You are copying the file during build-time and it is not on the user system as run-time.
Is this step really needed?